Sprint-2/myexc2.js

```diff
@@ -0,0 +1,59 @@
+/*function formatAs12HourClock() {}
+
+const currentOutput = formatAs12HourClock("08:00");
+const targetOutput = "08:00 am";
+console.assert(
+  currentOutput === targetOutput,
+  `current output: ${currentOutput}, target output: ${targetOutput}`
+);*/
+
+
+/*function formatAs12HourClock(time) {
+  return `${time} am`;
+}
+
+const currentOutput = formatAs12HourClock("09:00");
+const targetOutput = "09:00 am";
+
+console.assert(
+  currentOutput === targetOutput,
+  `current output: ${currentOutput}, target output: ${targetOutput}`
+);*/
+
+function formatAs12HourClock(time) {
```

Comment:
What would happen if the time was in the afternoon?

Reply:
It wouldn't work. Fix the function to the 12 hours format.